bin/rmt-mirror

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require_relative '../config/boot'
+require 'pathname'
+require 'config'
+require 'pp'
+
+require_relative '../lib/rmt'
+require_relative '../lib/rmt/mirror'
+require_relative '../lib/rmt/rpm'
+require_relative '../lib/rmt/downloader'
+
+repository_url = ARGV[0]
+local_path = ARGV[1]
+
+unless (repository_url and local_path)
```

Comment:
I guess later on it will read repos configuration from the DB. There's a different card about configuring which repos to mirror. For now it takes repo URL as parameter.